bin/react-generate.js

```diff
@@ -190,7 +194,7 @@ switch (commandLineArgs[0]) {
           shell.cd(cwd);
           directories.forEach(component => {
```

Comment:
let's call this `container`, just for clarification

```suggestion
          directories.forEach(container => {
```